assignments/lambda-classes.js

```diff
@@ -1 +1,114 @@
 // CODE here for your Lambda Classes
+class Person {
+	constructor(attr){
+		this.name = attr.name;
+		this.age = attr.age;
+		this.location = attr.location;
+	}
+
+	speak() {
+		return `Hello my name is ${this.name}, and I am from ${this.location}`;
+	}
+}
+
+class Instructor extends Person {
+	constructor(attr){
+		super(attr);
+		this.specialty = attr.specialty;
+		this.favLanguage = attr.favLanguage;
+		this.catchPhrase = attr.catchPhrase;
+	}
+
+	demo(subject){
+		return `Today we are learning about ${subject}.`;
+	}
+
+	grade(student, subject){
+		return `${student.name} receives a perfect score on ${subject}`;
+	}
+
+	gradeReview(student, pointsAdjust){
+		return student.grade = student.grade += Math.random() + pointsAdjust;
+	}
+}
+
+class Student extends Person {
+	constructor(attr){
+		super(attr);
+		this.previousBackground = attr.previousBackground;
+		this.className = attr.className;
+		this.favSubjects = attr.favSubjects;
+		this.grade = attr.grade;
+	}
+
+	listsSubjects(){ //Loop that console logs each favSubjects
+		for(var subject of this.favSubjects){
+			console.log(subject);
+		}
```

Comment:
Very nice, looping through and logging the individual `favSubjects` items here instead of just logging the array itself. 👍